.vitepress/theme/components/Feedback/Feedback.vue

```diff
@@ -82,78 +84,93 @@ watch(
 </script>
 
 <template>
-    <div class="feedbackContainer">
-
-      <form name="webcontainer-api-doc-feedback" data-netlify="true" hidden>
-        <textarea name="feedback"></textarea>
-        <input name="wasHelpful" />
-        <input name="page" />
-      </form>
-
-        <p class="title">Was this page helpful?</p>
-        <div class="buttonContainer">
-
-          <CardSingle :link="![FeedbackState.NO].includes(currentState)">
-            <button class="button"
-              :class="
-                [FeedbackState.YES].includes(currentState) && 'active',
-                [FeedbackState.NO].includes(currentState) && 'deselected'
-              "
-              @click="onButtonClick(FeedbackState.YES)"
-            >
-              <span class="icon thumbsUp"></span> Yes
-            </button>
-          </CardSingle>
-
-          <CardSingle :link="![FeedbackState.YES].includes(currentState)">
-            <button class="button"
-              :class="
-                [FeedbackState.NO].includes(currentState) && 'active',
-                [FeedbackState.YES].includes(currentState) && 'deselected'
-              "
-              @click="onButtonClick(FeedbackState.NO)"
-            >
-              <span class="icon thumbsDown"></span> No
-            </button>
-          </CardSingle>
-
-          <button v-if="[FeedbackState.YES, FeedbackState.NO].includes(currentState)" class="cancel" @click="cancelForm">Cancel</button>
-        </div>
-
-      <template v-if="[FeedbackState.YES, FeedbackState.NO].includes(currentState)">
-        <form
-          class="form"
-          name="doc-feedback"
-          method="post"
-          @submit.prevent="submitForm"
-          data-netlify="true"
+  <div class="feedbackContainer">
+    <form name="webcontainer-api-doc-feedback" data-netlify="true" hidden>
+      <textarea name="feedback"></textarea>
+      <input name="wasHelpful" />
+      <input name="page" />
+    </form>
```

Comment:
Whitespace formatting by Prettier is confusing GitHub's diff view quite a bit here.

The only changes in this component are about SVG icons:

```diff
-      <span class="icon thumbsUp"></span>
+      <SvgIcon class="thumbsUp" icon="fa-thumbs-up" size="12px" />
```

And some removed CSS in the `<style>` tag below.